libs/core/futures/tests/unit/local_use_allocator.cpp

```diff
@@ -32,4 +32,6 @@ int main()
                       test_allocator<int>>::value,
         "std::uses_allocator<local::packaged_task<int()>, "
         "test_allocator<int> >::value");
+
+    return 0;
```

Comment:
Not really necessary.

Reply:
I know :/ I usually add it for consistency anyways.

Reply:
I suspected you knew :P Maybe someone who didn't know before knows now... I don't mind keeping the explicit return.